lib/sequenceserver/database.rb

```diff
@@ -190,11 +192,22 @@ def make_blast_database?(file, type)
       # Returns user input if any. Auto-generated title otherwise.
       def get_database_title(path)
         default = make_db_title(File.basename(path))
-        print "Enter a database title or will use '#{default}': "
+        print "Enter a database title or will use #{default}: "
```

Comment:
This line has again been accidentally modified. Please revert to original (single quote around `#{default}`.